examples/ha-nat-gateway/README.md

```diff
@@ -21,9 +21,11 @@ terraform plan
 terraform apply
 ```
 
-SSH into the instance by hopping through one of the NAT gateway instances.
+SSH into the instance by hopping through one of the NAT gateway instances, first make sure that SSH agent is running and your private SSH key is added to the authentication agent.
 
 ```
+eval ssh-agent $SHELL
+ssh-add ~/.ssh/[YOUR_PRIVATE_KEY]
```

Comment:
Should we set this to the default ssh key that gcloud creates for you? 

    ~/.ssh/google_compute_engine

Reply:
Makes total sense, modified.